src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -319,13 +310,39 @@
         }
         this.isPending = false
       },
+      async removeLimits (limits) {
+        this.isPending = true
 
+        if (limits.id === 0) {
+          this.isPending = false
+          return false
+        }
```

Comment:
1. move this check to the top of the function, so you will no need to do `this.Pending = false` anymore
2. hide or disable the "remove" button if limits cannot be removed